lora-phy/src/sx1276_7_8_9/mod.rs

```diff
@@ -544,10 +549,35 @@ where
     }
 
     async fn get_rx_packet_status(&mut self) -> Result<PacketStatus, RadioError> {
-        let rssi_raw = self.read_register(Register::RegPktRssiValue).await?;
-        let rssi = (rssi_raw as i16) - 157i16; // or -164 for low frequency port ???
-        let snr_raw = self.read_register(Register::RegPktRssiValue).await?;
-        let snr = snr_raw as i16;
+        let snr = {
```

Comment:
Please include comment:
`// TODO: Fix for sx1272`

Reply:
Added the comment.